src/include/esp-idf/bindings.h

```diff
@@ -395,4 +395,16 @@
 
 #endif // CONFIG_BT_ENABLED
 
+// BLE Mesh
```

Comment:
LGTM, but if you could move this block of code **above** line 396. I.e. in the `CONFIG_BT_ENABLED`? Just in case... I'm not sure it is possible to have `CONFIG_BLE_MESH` enabled and then `CONFIG_BT_ENABLED` set to false (n), but better safe than sorry.